cypress/support/commands.ts

```diff
@@ -260,6 +260,26 @@ Cypress.Commands.add('updateChapter', (chapterId, data) => {
   };
   return cy.authedRequest(gqlOptions(chapterMutation));
 });
+/**
+ * Update event using GQL mutation
+ * @param eventId Id of the event
+ * @param data Data of the event. Equivalent of CreateEventInputs for the Events resolver.
+ */
+Cypress.Commands.add('updateEvent', (eventId, data) => {
+  const eventMutation = {
+    operationName: 'updateEvent',
+    variables: {
+      eventId,
+      data: { ...data },
```

Comment:
```suggestion
      data,
```
We don't need a copy, the original object is fine here.